regen-network/regen-web
# feat: toucan project and batch pages

## Description

Closes: regen-network/regen-registry#1307
Closes: regen-network/regen-registry#1401



- Merges Project Page Metadata components (VCS vs CFC) into single component, `ProjectPageMetaData`.
- Merges Batch Metadata components into single component, `BatchMetadataAdditionalInfo`
- Adds Toucan-related types and fields
- Applies some style tweaks to meet recent mockups and https://monosnap.com/file/YnoRIMFgZRzLcawmWFY01lJRIDoA6O
- In Metadata components, displays any of the fields we might expect _IF_ they are available in the IRI-resolved metadata.

---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...


#### Setting up backport PRs

After merging your PR to `dev`, set up backports by doing the following:

1. If your branch does not have merge commits, add the following comment to
   your PR:

- If PR is a new feature: `@Mergifyio backport redwood hambach`.
- If PR is a bug fix: `@Mergifyio backport redwood hambach master`.

2. If your branch does have merge commits:

   a. Pull latest `dev`, `hambach` and `redwood` branches

   b. Create new branches for backports and merge `dev` (replace `<PR#>` with your PR #)

   ```
   git checkout -b hambach-backport-<PR#> hambach
   git merge dev
   git push origin hambach-backport-<PR#>
   git checkout -b redwood-backport-<PR#> redwood
   git merge dev
   git push origin redwood-backport-<PR#>`
   ```

   c. Open new PRs in regen-web targeting `hambach` and `redwood`, respectively.

   d. If PR is a bug fix repeat the same process for `master` branch.

### How to test

1. https://deploy-preview-1588--regen-registry.netlify.app/projects/C46-001
2. https://deploy-preview-1588--regen-registry.netlify.app/credit-batches/C46-001-20140701-20141231-001
    - Note: We need to update the metadata for this before we see fields in "Additional Info" (access requested)
3. Regression (optional): Make sure other project and batch pages' "Additional Info" look the same

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**._

I have...
